src/main/java/com/devonfw/quarkus/productmanagement/domain/model/ProductEntity.java

```diff
@@ -1,30 +1,37 @@
-package com.devonfw.demoquarkus.domain.model;
+package com.devonfw.quarkus.productmanagement.domain.model;
+
+import java.math.BigDecimal;
 
 import javax.persistence.Entity;
 import javax.persistence.GeneratedValue;
 import javax.persistence.Id;
 import javax.persistence.Table;
 
+import com.devonfw.quarkus.productmanagement.ApplicationPersistenceEntity;
+
 import lombok.Getter;
 import lombok.Setter;
 
 @Getter
 @Setter
 @Entity
-@Table(name="Animal")
+@Table(name = "Product")
 // A JPA entity requires at least 2 things @Entity annotation and an ID
 // by default, the DB table will have the same name as our class
-public class AnimalEntity {
+public class ProductEntity extends ApplicationPersistenceEntity {
 
   @Id
   @GeneratedValue
-  private Long id;
+  // private Long id;
```

Comment:
Remove the outcommented lines. You can also remove the `@Id` and `@GeneratedValue` as they are no longer needed in this class as they are now included in `ApplicationPersistenceEntity`.